airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/InlineEnrollmentCallout.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { PropsWithChildren } from "react";
+import { FormattedMessage } from "react-intl";
+
+import { Callout } from "components/ui/Callout";
+import { Text } from "components/ui/Text";
+
+import styles from "./InlineEnrollmentCallout.module.scss";
+
+export const EnrollLink: React.FC<PropsWithChildren<unknown>> = ({ children }) => {
+  const onEnrollClick = () => {
+    return null;
+  };
+
+  return (
+    <button onClick={onEnrollClick} className={styles.enrollLink}>
+      {children}
+    </button>
```

Comment:
Also think it's fine in this case. I do think we should have a reusable component for this in the component library so that we don't have to repeat the `.enrollLink` CSS. But we don't have to address that here!

Reply:
https://github.com/airbytehq/airbyte/issues/18825